### DISCUSSION TITLE
I have an Asus Tuf f15 (fx506hf) .I disabled hyperthreading to increase my battery life , does it impacts the g helper modes

### DISCUSSION THREAD

**Discussion by GXhacker12** [seerge/g-helper#2888] [General]
I have an Asus Tuf f15 (fx506hf).I disabled hyperthreading to increase my battery life , does it impacts the g helper modes

**Comment by seerge** [comment]
@GXhacker12 hello, modes are stored in BIOS, not in G-Helper. I didn't completely get what and where did you disable, but I can imagine it will affect CPU performance.

**Comment by DZXTRICKS** [comment]
Hey anyway did your un-hyperthreading does something to the battery life? I find no practical difference (and almost no performance changes as well)
